ouroboros-consensus/src/Ouroboros/Consensus/Node.hs

```diff
@@ -115,6 +115,9 @@ data RunNodeArgs blk = RunNodeArgs {
       -- | Network magic
     , rnNetworkMagic :: NetworkMagic
 
+      -- | Initiator only flag which disables network duplex mode.
+    , rnInitiatorOnly :: InitiatorOnly
+
```

Comment:
This flag should also disable the running of the responder in Diffusion.hs.

Reply:
I'll add this.